index.html

```diff
@@ -48,8 +50,166 @@ <h3 class="heading-3">Let's connect</h3>
       <a href="#">
         <img class="social-icons" src="./images/fingure-img.png" alt="fingure icon">
       </a>
+      </div>
     </section>
+    <!-----===== Work section ======---->
 </section>
-  </header>
+<main> 
+<div class="card-container">
+  <div class="work-section">
+    <img class="cards" src="./images/snap-card-4.png" alt="avocode">
+    <h2 class="heading-2">Tonic</h2>
+
+    <div class="specify">
+    <p class="card-text-1">Canopy</p>
+    <img class="dot" src="./images/dot.png" alt="dot img">
+    <p class="card-text">Back End Dev</p>
+    <img class="dot"  src="./images/dot.png" alt="dot img">
+    <p class="card-text">2015</p>
+    </div>
+
+    <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+    </p>
+    <ul class="languages">
+    <li class="lang" >html</li>
+    <li class="lang" >css</li>
+    <li class="lang">javascript</li>
+    </ul>
+    <button class="btn">See project</button>
```

Comment:
- [ ] There are some issues in your buttons, kindly consider applying following changes in all the buttons in your project to make it similar to the figma design which is a project requirement.
    -  the background of your buttons is not white as in the figma design. consider making it similar to the figma design.
    -  the hover state background color is not as suggested in the figma design. consider changing it also.
    -  In the figma design, there are button guidelines for the pressed state. you can achieve that by applying that background color with `:active` pseudoclass on your button.
 your button design
![qw](https://user-images.githubusercontent.com/96683843/219850288-273193a6-2f23-4571-bd58-c6a86d2600ac.jpg)

figma design guidelines
![zx](https://user-images.githubusercontent.com/96683843/219850290-58b3c27f-bdf6-4c66-b28f-e8fe22e42577.jpg)